exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/LogAdapter.java

```diff
@@ -96,6 +94,14 @@ private static InstrumentationLibraryLogs buildInstrumentationLibraryLogs(
             .setSpanId(
                 UnsafeByteOperations.unsafeWrap(
                     OtelEncodingUtils.bytesFromBase16(logRecord.getSpanId(), SpanId.getLength())));
+    String severityText = logRecord.getSeverityText();
```

Comment:
Actual potential NPE

Reply:
proto classes will NPE if passed a null? 😭

Reply:
Proto3 never returns any nulls from getters for primitives (even for strings that are never set, you get empty strings), so it makes sense that it would also reject setting them. I have to admit that I was bitten by that too 😄